chronos/internal/asyncfutures.nim

```diff
@@ -1667,8 +1667,7 @@ proc `or`*[T, Y, E1, E2](
   type
     InternalRaisesFutureRaises = union(E1, E2)
```

Comment:
this might be a problem if neither of E1, E2 raise `CancelledError`

Reply:
How i can do this? Because looks like `prepend` and `union` are not working here.

Reply:
Fixed in https://github.com/status-im/nim-chronos/pull/517/commits/d5fd6663b2997328fda2a63aff9c9f69955c5b3f